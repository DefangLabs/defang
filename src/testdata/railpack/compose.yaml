services:
  railpack-short:
    build: .
    command: npm start
  railpack-long:
    build:
      context: .
    command: exec node index.js
  railpackwithdockerfile:
    build:
      context: .
      dockerfile: Dockerfile
    command: something "with space"
