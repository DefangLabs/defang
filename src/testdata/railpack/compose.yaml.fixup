{
  "railpackwithdockerfile": {
    "build": {
      "context": ".",
      "dockerfile": "Dockerfile"
    },
    "command": null,
    "entrypoint": null,
    "networks": {
      "default": null
    }
  },
  "wrailpack-long": {
    "build": {
      "context": ".",
      "dockerfile": "Dockerfile"
    },
    "command": null,
    "entrypoint": null,
    "networks": {
      "default": null
    }
  },
  "wrailpack-short": {
    "build": {
      "context": ".",
      "dockerfile": "Dockerfile"
    },
    "command": null,
    "entrypoint": null,
    "networks": {
      "default": null
    }
  }
}