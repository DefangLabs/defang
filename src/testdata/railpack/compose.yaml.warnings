service "railpackwithdockerfile": dockerfile not found: dockerfile not found
service "wrailpack-long": dockerfile not found: dockerfile not found
service "wrailpack-short": dockerfile not found: dockerfile not found
