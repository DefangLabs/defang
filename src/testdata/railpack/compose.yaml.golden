name: railpack
services:
  railpack-long:
    build:
      context: .
      dockerfile: Dockerfile
    command:
      - exec
      - node
      - index.js
    networks:
      default: null
  railpack-short:
    build:
      context: .
      dockerfile: Dockerfile
    command:
      - npm
      - start
    networks:
      default: null
  railpackwithdockerfile:
    build:
      context: .
      dockerfile: Dockerfile
    command:
      - something
      - with space
      - ${PORT}
    networks:
      default: null
networks:
  default:
    name: railpack_default
